rackcity/views/it_instance_views.py

```diff
@@ -533,7 +532,10 @@ def instance_page_count(request):
     Return total number of pages according to page size, which must be
     specified as query parameter.
     """
-    if not request.query_params.get('page_size') or int(request.query_params.get('page_size')) <= 0:
+    if (
+        not request.query_params.get('page_size')
+        or int(request.query_params.get('page_size')) <= 0
+    ):
```

Comment:
no code changes just line length linting ;    )

Reply:
:     )

Reply:
I tried to make that smiley face longer but it wouldn't let me

Reply:
:         )

Reply:
same